test/common/page_objects/search-page.js

```diff
@@ -22,6 +22,7 @@ class FindPage {
       summary: this.extractText('summary', context),
```

Comment:
I've changes this on some previous PR. after master rebase this is looking good now.